src/main/java/com/skcraft/plume/module/Status.java

```diff
@@ -0,0 +1,54 @@
+package com.skcraft.plume.module;
+
+import com.sk89q.intake.Command;
+import com.sk89q.intake.Require;
+import com.skcraft.plume.command.Sender;
+import com.skcraft.plume.common.util.module.Module;
+import com.skcraft.plume.util.Messages;
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraftforge.event.entity.living.LivingAttackEvent;
+
+import java.util.HashSet;
+import java.util.UUID;
+
+import static com.skcraft.plume.common.util.SharedLocale.tr;
+
+@Module(name = "status", desc = "Heal, feed and god commands")
+public class Status {
+
+    HashSet<UUID> gods = new HashSet<>();

```

Comment:
I would make `gods` private and also define it as a `Set<UUID>`. This also does constitute a (minor) memory leak, although solving that problem is an involved process for which there is nothing in Plume to handle that with yet.